src/content/en/tools/workbox/guides/advanced-recipes.md

```diff
@@ -173,3 +173,44 @@ worbox.routing.registerRoute(new RegExp('/images/'), ({event}) => {
     .catch(() => caches.match(FALLBACK_IMAGE_URL));
 });
 ```
+
+## Use postMessage() to notify of cache updates
+
+Not every browser which supports service workers also supports the
+[Broadcast Channel API](https://developer.mozilla.org/en-US/docs/Web/API/Broadcast_Channel_API).
+On browsers which lack [Broadcast Channel API support](https://caniuse.com/#search=broadcast), the
+`workbox-broadcast-cache-update` plugin will not send any messages letting clients know that a
+cached response was updated.
+
+As an alternative to using `workbox-broadcast-cache-update`, you can instead "roll your own" plugin
+which listens for the same cache update events, and uses the more widely supported [`postMessage()`
+API](https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope/postMessage) for
+sending out notifications. In this custom plugin, you have control over what criteria are
+used to determine whether a cached response has been updated. You can use whatever message format
+inside of `postMessage()` that makes sense for your use case.
+
+Here's an example of one possible implementation:
+
+```javascript
+const postMessagePlugin = {
+  cacheDidUpdate: async ({cacheName, url, oldResponse, newResponse}) => {
+    // Use whatever logic you want to determine whether the responses differ.
+    if (oldResponse && (oldResponse.headers.get('etag') !== newResponse.headers.get('etag'))) {
+      const clients = await self.clients.matchAll();
+      for (const client of clients) {
+        // Use whatever message body makes the most sense.
+        // Note that `Response` objects can't be serialized.
+        client.postMessage({url, cacheName});
```

Comment:
might want this in the Flux format to match what you do for `BroadcastChannel`.  Up to you though just throwing it out there :) thats what we did in our plugin

Reply:
I'd like to just leave it open for folks to add in whatever data they need.